packages/react-dom-bindings/src/events/ReactDOMEventListener.js

```diff
@@ -57,7 +57,7 @@ import {isRootDehydrated} from 'react-reconciler/src/ReactFiberShellHydration';
 const {ReactCurrentBatchConfig} = ReactSharedInternals;
 
 // TODO: can we stop exporting these?
-export let _enabled: boolean = true;
```

Comment:
This is just an unnecessary export I saw. It's the reason we end up with dead code that the linter doesn't detect in many cases and why I'm particular about looking for unused exports in code reviews.